future/name-resolution/recursive-dns/README.md

```diff
@@ -623,30 +623,47 @@ Would you look at that! We got our whole lookup done now! Congratulations team!
 
 NOTE: We hacked this together a little... We used glue records in the zonefile we created for `meow.` to point our resolvers to the authoritative servers. But those actual records don't exist anywhere in our DNS. We still need to go back and add actual records for `authoritative-a.aws.meow.` and `authoritative-s.supercorp.meow.`. We'll leave that as an exercise for the reader.
 
-# Exercises
+## Watch the recursive DNS lookup using `tcpdump`
 
-* watching tcp dump of name resolution
+<!-- TODO -->
```

Comment:
Do you want to leave this section in this PR or take it all out until you've got it done?

Reply:
Ah! excellent catch. Hmmm... If you're down, I'd like to merge this PR and then make a new one specifically for reading the tcpdump output. Does that sound OK to you?